src/qt/lookupspdialog.h

```diff
@@ -39,6 +41,9 @@ public Q_SLOTS:
     Ui::LookupSPDialog *ui;
     WalletModel *model;
 
+    GUIUtil::TableViewLastColumnResizingFixer *borrowedColumnResizingFixer;
+    void addDenominationRow(uint8_t id, const std::string& value);
```

Comment:
We should put this one in a separated group instead of mixing with variable group.